test/LndClient/RPCSpec.hs

```diff
@@ -442,8 +417,10 @@ spec = around withEnv $ do
       ]
     somePubKey env = do
       res <- runApp env $ coerceRPCResponse =<< getPeers (envLnd env)
-      let peersList = head $ peers res
-      return $ pubKey peersList
+      let mPeer = safeHead $ peers res
+      case mPeer of
```

Comment:
not a big deal, but there is standard functions called `maybe` which can be used there to avoid case expression
there is also `fromMaybe` which can be used in more simper cases
this can be refactored later